pkg/platform/api/buildplanner/model/buildplan.go

```diff
@@ -54,8 +54,18 @@ const (
 
 	VersionRequirementComparatorKey = "comparator"
 	VersionRequirementVersionKey    = "version"
+
+	XArtifactMimeType            = "application/x.artifact"
+	XActiveStateArtifactMimeType = "application/x-activestate-artifacts"
+	XCamelInstallerMimeType      = "application/x-camel-installer"
 )
 
+func IsStateToolArtifact(mimeType string) bool {
+	return mimeType == XArtifactMimeType ||
+		mimeType == XActiveStateArtifactMimeType ||
+		mimeType == XCamelInstallerMimeType
```

Comment:
Are we sure this is a valid state tool artifact? IIRC we actually have offline installers for camel, just making sure that's not what we're matching here.

Reply:
Yes, in the camel builds the artifacts with the mime type of `application/x-camel-installer` are what we should be installing. Here's an example of the artifact listing from a camel build plan

```
{
    "artifacts": [
        {
            "__typename": "ArtifactSucceeded",
            "nodeId": "3863e868-156c-5efb-8cc6-f15e961478ae",
            "mimeType": "application/x-activestate-builder",
            "generatedBy": "9792f42a-69f2-5577-b947-51c9b0570d93",
            "runtimeDependencies": [],
            "status": "SUCCEEDED",
            "logURL": "",
            "url": "s3://INVALID",
            "checksum": "816a927e49b8a5484d059afda25a1252517277f0"
        },
        {
            "__typename": "ArtifactSucceeded",
            "nodeId": "6fbb595e-ec6d-5347-ac9c-0c770b36472b",
            "mimeType": "text/plain",
            "generatedBy": "a1062e9b-b120-505e-900d-04c6aac1dac5",
            "runtimeDependencies": [],
            "status": "SUCCEEDED",
            "logURL": "https://s3.amazonaws.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.log",
            "url": "https://dl.activestate.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e-packages.txt",
            "checksum": ""
        },
        {
            "__typename": "ArtifactSucceeded",
            "nodeId": "27f76f04-5bf4-5091-851f-4d6ff0bfffc1",
            "mimeType": "application/gzip",
            "generatedBy": "a1062e9b-b120-505e-900d-04c6aac1dac5",
            "runtimeDependencies": [],
            "status": "SUCCEEDED",
            "logURL": "https://s3.amazonaws.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.log",
            "url": "https://dl.activestate.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e-tests.tar.gz",
            "checksum": ""
        },
        {
            "__typename": "ArtifactSucceeded",
            "nodeId": "5cdc504a-95d5-53c7-95c3-9605d468dbf0",
            "mimeType": "application/x-xar",
            "generatedBy": "a1062e9b-b120-505e-900d-04c6aac1dac5",
            "runtimeDependencies": [],
            "status": "SUCCEEDED",
            "logURL": "https://s3.amazonaws.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.log",
            "url": "https://dl.activestate.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.pkg",
            "checksum": ""
        },
        {
            "__typename": "ArtifactSucceeded",
            "nodeId": "58efa9c2-e49d-527c-a7d2-4adea0d410e3",
            "mimeType": "application/x-camel-installer",
            "generatedBy": "a1062e9b-b120-505e-900d-04c6aac1dac5",
            "runtimeDependencies": [],
            "status": "SUCCEEDED",
            "logURL": "https://s3.amazonaws.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.log",
            "url": "https://dl.activestate.com/camel-builds/ActivePython/darwin-17.7.0/20200320T153029Z/ActivePython-2.7.16.0000-macosx10.9-x86_64-816a927e.tar.gz",
            "checksum": ""
        }
    ]
}
```